src/splitseq_mkref.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Change to `#!/usr/bin/bash` or `#!/usr/bin/env bash`.

Reply:
Will do.